### DISCUSSION TITLE
Entity framework documentation?

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [ChilliCream/hotchocolate#5502] [Q&A]
Hi I just gave HotChocolate a spin and great work it works like a charm!

But the documentation seems to lack the most useful parts is there a reason for this?
The following [page](https://chillicream.com/docs/hotchocolate/integrations/entity-framework) doesn't mention anything about 
```C#
    .AddProjections()
    .AddFiltering()
    .AddSorting()
    .AddQueryableCursorPagingProvider()
```

and
```C#
    [UseDbContext(typeof(SchoolContext)),
     UsePaging(MaxPageSize = 100, IncludeTotalCount = true),
     UseProjection,
     UseFiltering,
     UseSorting]
    public IQueryable<Student> AllStudents([Service] SchoolContext context) => context.Students;
```

Is there a reason for this? seems like the code above removes 99% of all boiler plate code I would have to write otherwise?

**Comment by aaron-manning** [comment]
Agree, it was difficult to discover these features
```
    .AddProjections()
    .AddFiltering()
    .AddSorting()
    .AddQueryableCursorPagingProvider()
```

Sometimes it's useful to intercept/modify the query sent by the client, so that you can modify the final iQueryable returned.
I hope HC 13 comes with comprehensive docs on these features as it's a current pain point.
